.travis.yml

```diff
@@ -26,4 +26,4 @@ matrix:
     - rvm: jruby-head
     - rvm: 1.9.3
 
-script: "rake compile && bundle exec rspec --color --backtrace --tag ~unfinished --seed 1 --format documentation ./spec"
+script: "rake compile && bundle exec rspec --color --backtrace --tag ~unfinished --seed 1 ./spec"
```

Comment:
I think the `--format documentation` was there to see the last test where it was hanging. Is there another way?